common/src/types/v0/transport/replica.rs

```diff
@@ -21,6 +22,103 @@ impl GetReplicas {
     }
 }
 
+/// Pool on which the resides
+#[derive(Serialize, Default, Debug, Clone, Eq, PartialEq)]
+pub struct ReplicaPool {
+    /// name of the pool
+    pub name: PoolId,
+    /// uuid of the pool
+    pub uuid: Option<PoolUuid>,
+}
+
+impl ReplicaPool {
+    /// Return a new `Self` from the provided arguments
+    pub fn new(name: &PoolId, uuid: Option<PoolUuid>) -> Self {
+        Self {
+            name: name.clone(),
+            uuid,
+        }
+    }
+    /// get the uuid of the pool
```

Comment:
We have been following this  :- `writing doc comments starting with Capital and ending with period`.